feature/office/src/main/java/com/iteneum/office/data/OfficeModel.kt

```diff
@@ -1,8 +1,5 @@
 package com.iteneum.office.data
 
-import com.itenum.domain.ContactTelephone
-import com.itenum.domain.Email
-
 /**
  * This class is for the data for each item in the list.
  * @param address the address of the busines
```

Comment:
in param address change busines to business

Reply:
changed the typo
![imagen](https://github.com/irv205/droid_leaspert/assets/128720218/f773d2ab-6fa6-403c-8d5d-625751f9893b)